about/glossary.rst

```diff
@@ -1,58 +0,0 @@
-========
-Glossary
-========
-
-.. glossary::
-
-   AWS
-       `Amazon Web Services <https://aws.amazon.com/>`_ offers reliable, scalable, and inexpensive cloud computing services.
-      Free to join, pay only for what you use.
-
-   Linode
-      `Linode.com <https://www.linode.com/>`_ is an American privately owned virtual private server provider company
-       based in Galloway, New Jersey, United States.
-
-   DigitalOcean
-      `DigitalOcean, Inc. <https://www.digitalocean.com/>`_ is an American cloud infrastructure provider
-       headquartered in New York City with data centers worldwide.
-
-   ZODB
-      `A native object database for Python <http://www.zodb.org/en/latest/>`_.
-
-   TTW
-      Through-The-Web, changes in the browser/
-
-   S3
-      `Amazon S3 <https://aws.amazon.com/s3/>`_ - Object storage built to store and retrieve any amount of data from anywhere.
-
-   NFS
-      `Network File System <https://en.wikipedia.org/wiki/Network_File_System>`_.
-
-   Amazon Opsworks
-      `AWS OpsWorks <https://aws.amazon.com/opsworks/>`_ is a configuration management service that uses Chef,
-       an automation platform that treats server configurations as code.
-
-   Ansible
-      `Ansible <https://www.ansible.com/>`_ is an open source automation platform.
-       Ansible can help you with configuration management, application deployment, task automation.
-
-   Chef
-      `A configuration management tool written in Ruby and Erlang <https://www.chef.io/products/chef-infra/>`_.
-
-   CloudFormation
-      `AWS CloudFormation <https://aws.amazon.com/cloudformation/>`_ gives developers and systems administrators an way to create and manage
-      a collection of related AWS resources, provisioning and updating them in an orderly and predictable fashion.
-
-   Travis CI
-      Travis CI is a hosted, distributed continuous integration service used to build and test software projects hosted at GitHub.
-      Open source projects may be tested at no charge via `travis-ci.org <https://travis-ci.org>`_.
-
-   Solr
-      `Solr <http://lucene.apache.org/solr/>`_ is a popular, blazing-fast, open source enterprise search platform built on Apache Lucene.
-
-   ZCML
-      The `Zope Configuration Mark-up Language <https://docs.plone.org/develop/addons/components/zcml.html>`_.
-
-   Diazo
-      `Diazo theme engine guide <http://docs.diazo.org/en/latest/>`_
-      Diazo allows you to apply a theme contained in a static HTML web page to a dynamic website created using any server-side technology.
```

Comment:
This and all other `.rst` files are removed in #512. You will need to rebase on `master`, once that PR is merged.